arm9/source/filesys/fsutil.c

```diff
@@ -818,8 +819,8 @@ bool FileSelectorWorker(char* result, const char* text, const char* path, const
                     char* dot = strrchr(temp_str, '.');
                     if (dot) *dot = '\0';
                 }
-                ResizeString(opt_names[n_opt], temp_str, 32 - 6, 8, false);
-                if (entry->type == T_DIR) snprintf(opt_names[n_opt] + 32 - 6, 6 + 1, " (dir)");
+                ResizeString(opt_names[n_opt], temp_str, _MAX_FS_OPT_LEN - ((entry->type == T_DIR) ? 6 : 0), 8, false);
```

Comment:
You're changing this because, otherwise, there would be leftovers from the line above / below when scrolling.

Reply:
Okay.